services/node_stake_update.proto

```diff
@@ -112,7 +112,7 @@ message NodeStake {
   int64 reward_rate = 4;
 
   /**
-   * Consensus weight of this node for the new staking period.
+   * Total stake (rewarded and not rewarded) of this node at the beginning of the new staking period.
```

Comment:
it's true only when  min_stake <= stake_rewarded + stake_not_rewarded <= max_stake I believe. i.e. if the sum is less than min_stake, stake is 0; if the sum is more than max_stake, stake = max_stake

Reply:
Yeah just discussed with Nick and the same field can be used as the `consensus weight` .